### PR TITLE
add status and insync badge to subscription detail page

### PR DESCRIPTION
**Pull request by wouter1975**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+export type WfoSubscriptionSyncStatusBadgeProps = {
+    insync: boolean;
+};
+
+export const WfoSubscriptionSyncStatusBadge: FC<
+    WfoSubscriptionSyncStatusBadgeProps
+> = ({ insync }) => {
+    const { theme, toSecondaryColor } = useOrchestratorTheme();
+
+    const getBadgeColorFromStatus = (insync: boolean) => {
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx:15]: This returning more than the color so I would call it getBadgeProperties or something

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx:1]: Add a storybook story. See other badges for an example

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            return {
+                badgeColor: toSecondaryColor(success),
+                textColor: successText,
+                insyncText: 'in-sync',
+            };
+        } else {
+            return {
+                badgeColor: toSecondaryColor(danger),
+                textColor: dangerText,
+                insyncText: 'out-of-sync',
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx:28]: Double check. These don't need to be translated right?

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx (hunk 2 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx:1]: Added

**Pull request merged**